test/distributed/org/apache/cassandra/fuzz/harry/examples/RangeTombstoneBurnTest.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.fuzz.harry.examples;
```

Comment:
Package naming and directory structure ideas...

1.) Taking inspiration from the simulator, it feels like we could probably just simplify `org.apache.cassandra.fuzz.harry` -> `org.apache.cassandra.harry` unless we think there will be a proliferation of other fuzz testing tools.

2.) The `test` -> `fuzz` folder structure could also just be `test` -> `harry`, or at least have a `harry` sub-folder, given the next directory down is just `main`.

3.) There are a bunch of tests in the `distributed` folder that are really just unit tests for Harry. It feels like it would make more sense to add a `test` folder alongside `main` (and the actual Harry code) for those tests, much like what Simulator does. Even things like `RangeTombstoneBurnTest` could move. We just have to draw a clear line separating things that test Harry and tests for database components that happen to use the Harry library.